README.md

```diff
@@ -4,10 +4,14 @@
 # To Do Application
 - To Do List Manager Phase 1: Begin the build of an application designed for creating and organizing a “To Do” list in your browser.
 - To Do List Manager Phase 2: Connecting the To Do Application to an external API and Database
+- To Do List Manager Phase 3: Add some configuration settings to the application
+- To Do List Manager Phase 4: Adding security and access controls to the application.
 
 ## About Process
 - In the first phase, we’re going to have to start by converting a legacy application into a more modern architecture. Our initial “Proof of Concept” was written using class based components and was not properly styled. Now that our client has given us the “green light” for development, we’re going to refactor the application using Hooks and upgrading the style. 
-- In the second phase, we’ll be connecting our application to an API so that our list will be saved permanently. This will require a working and deployed API server with a proper “To Do” data model and appropriate REST routes
+- In the second phase, we’ll be connecting our application to an API so that our list will be saved permanently. This will require a working and deployed API server with a proper “To Do” data model and appropriate REST routes.
+- In Phase 3, we’d like to extend the functionality of the application by potentially allowing users to set a few configuration options to change how the application behaves. The user stories from Phases 1 and 2 remain unchanged. For this phase, we are now adding the following new user stories.
+- In this final phase, we’ll be requiring that users be logged in, in order to see the to do items. Additionally, based on their user type, they will be allowed (or denied) to perform actions such as editing or deleting them.
 
 ## Netlify Deployment
```

Comment:
- Would like to be able to delete/edit a task
- Would like to signup added to this and ACL added